Makefile

```diff
@@ -43,8 +43,8 @@ else
 endif
 
 export SELF_SIGNED_CERTS   ?= true
-export INSTALLATION_TYPE   ?= managed
-export INSTALLATION_NAME   ?= rhmi
+export INSTALLATION_TYPE   ?= managed-api
+export INSTALLATION_NAME   ?= managed-api
```

Comment:
I think we should stick with defaulting to 'rhmi' at this point. Defaulting to managed-api would require some README.md updates, changing the INSTALLATION_PREFIX default value and (likely) more.

Reply:
@trepel Is that in relation to the type, name or both?

Reply:
@philbrookes You specifically mentioned changing the installation name, do you have a strong opinion on this?

Reply:
Both. I definitely agree that Manged API should be default in the end, I am just not sure if everything has to be addressed in one big PR. I leave that up to you guys :)

Just make sure installation of RHMI is also part of verification steps.

Reply:
Sorry, I missed this ping. I think the default behaviour of the operator should be RHMI, it should only do managed-api when we pass in the env var for installation type.